### PR TITLE
add export/import logs from file, HexDisplay component, additional UI/UX Improvements

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

File: src/js/components/LogsPanel/LogsPanel.jsx [hunk 1]
```diff
@@ -91,8 +97,39 @@ const LogsPanel = ( {logs, setLogs} ) => {
         [copyToClipboard]
     );
 
-    const handleLogsLimitExceededDialogClose = () => {
-        setLogsLimitExceededDialogOpen(false);
+    const handleExportLogs = () => {
+        const now = new Date();
+        const formattedDate = now.toISOString().replace(/[:.]/g, '-');
```
Comment (DarkPark) [inline src/js/components/LogsPanel/LogsPanel.jsx:102]: not ISO

File: src/js/components/Log/utils/createSubLogTitle.jsx [hunk 2]
```diff
@@ -20,7 +21,8 @@ const createSubLogTitle = ( logCommand, commandType ) => (
                         ? (
                             <>
                                 {'id: '}
-                                <HighlightedText isMonospacedFont={true}>{getHexFromNumber(logCommand.command.id)}</HighlightedText>
+                                <HexDisplay hex={getHexFromNumber(logCommand.command.id)}/>
```
Comment (DarkPark) [inline src/js/components/Log/utils/createSubLogTitle.jsx:24]: BinaryViewer?

File: src/js/components/LogsPanel/LogsPanel.jsx (hunk 1 above)
Comment (aleks-and) [reply src/js/components/LogsPanel/LogsPanel.jsx]: updated it to use local time

File: src/js/components/Log/utils/createSubLogTitle.jsx (hunk 2 above)
Comment (aleks-and) [reply src/js/components/Log/utils/createSubLogTitle.jsx]: discussed, rename to HexViewer

**Review by DarkPark — APPROVED** [review]

**Review by DeadByElpy — APPROVED** [review]

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**